io.catenax.transmission.transmission_pass/1.0.0/TransmissionPass.ttl

```diff
@@ -0,0 +1,452 @@
+    bamm:dataType xsd:decimal;
+    bamm-c:unit unit:kilowatt.
+:TransmissionSpreading a bamm-c:Quantifiable;
+    bamm:preferredName "Transmission Spreading"@en;
+    bamm:description "The transmission spreading coefficient. The ratio between the smallest (shortest gear) and the largest (longest gear) ratio as decimal with no unit."@en;
+    bamm:dataType xsd:decimal.
+:Capacity a bamm-c:Quantifiable;
+    bamm:preferredName "Capacity"@en;
+    bamm:description "The oil capacity in dm³."@en;
+    bamm:dataType xsd:decimal;
+    bamm-c:unit unit:cubicDecimetre.
+:dismantlingProcedure a bamm:Property;
+    bamm:preferredName "dismantling procedure"@en;
+    bamm:description "Document containing the transmission dismantling procedure."@en;
+    bamm:characteristic :DocumentsCharacteristic.
+:vehicleDismantlingProcedure a bamm:Property;
+    bamm:preferredName "vehicle dismantling procedure"@en;
+    bamm:description "Document containing the dismantling procedure from the vehicle."@en;
+    bamm:characteristic :DocumentsCharacteristic.
+:packagingInstructions a bamm:Property;
+    bamm:preferredName "packaging instructions"@en;
+    bamm:description "Instructions for safely packaging of the transmission."@en;
+    bamm:characteristic :DocumentsCharacteristic.
+:transportationInstructions a bamm:Property;
+    bamm:preferredName "transportationInstructions"@en;
+    bamm:description "Instructions for safely transporting of the transmission."@en;
+    bamm:characteristic :DocumentsCharacteristic.
+:safetyMeasures a bamm:Property;
+    bamm:preferredName "safety measures"@en;
+    bamm:description "Includes all safety relevant documents for e.g. Dismantling."@en;
+    bamm:characteristic :DocumentsCharacteristic;
+    bamm:see <https://eclass.eu/eclass-standard/content-suche/show?tx_eclasssearch_ecsearch%5Bcc2prdat%5D=0173-1%2302-AAO190%23001&tx_eclasssearch_ecsearch%5Bdischarge%5D=0&tx_eclasssearch_ecsearch%5Bid%5D=-1&tx_eclasssearch_ecsearch%5Blanguage%5D=1&tx_eclasssearch_ecsearch%5Boption%5D=pr&tx_eclasssearch_ecsearch%5Bsearchtxt%5D=0173-1%2302-AAO190%23001&tx_eclasssearch_ecsearch%5Bversion%5D=12.0&cHash=ecd65dc92a134cdb1453556e4a1557ca>.
+:SparePartSupplier a bamm-c:List;
+    bamm:preferredName "SparePartSupplier"@en;
+    bamm:description "Characteristic for details of the  spare parts and suppliers of replacement parts / spare parts."@en;
+    bamm:dataType :SparePartEntity.
+:SparePartEntity a bamm:Entity;
+    bamm:preferredName "Spare Part Entity"@en;
+    bamm:description "Entity for details of the  spare parts and suppliers of replacement parts / spare parts."@en;
+    bamm:properties (:supplierContact :supplierId :sparePartName).
+:SubstancePercentage a bamm-c:Quantifiable;
+    bamm:preferredName "Substance Percentage"@en;
+    bamm:description "Substance percentage of the recycled content."@en;
+    bamm:dataType xsd:decimal;
+    bamm-c:unit unit:percent.
+:cobalt a bamm:Property;
```

Comment:
Is it really the case that those three substances need to be mentioned separately.
Maybe you could also use a list with datatype entity, with the entity having the per cent value and a property with a value from an entity referring the substance. than you could handle cobalt, lithium and nickel the same way as all other substances

Reply:
okay, aber diese drei sind Pflichtangaben, alle weiteren Werte sind optional. Ist das so mglich?